CMakeLists.txt

```diff
@@ -192,6 +193,21 @@ if(AMR_WIND_ENABLE_HELICS)
   target_compile_definitions(${amr_wind_lib_name} PRIVATE AMR_WIND_USE_HELICS)
 endif()
 
+if(AMR_WIND_TEST_WITH_PYTHON)
+  set(CMAKE_PREFIX_PATH ${PYTHON_DIR} ${CMAKE_PREFIX_PATH})
+  find_package(Python3 REQUIRED COMPONENTS Interpreter)
+  # Developer note: modify this list if modifying scripts used in python-based regression tests
+  set(PYTHON_REQUIREMENTS numpy netCDF4 matplotlib pathlib pandas)
+  foreach(pymod IN LISTS PYTHON_REQUIREMENTS)
+    execute_process(COMMAND pip3 show ${pymod}
```

Comment:
instead of using pip here, could we capture the exit code of running `python -c "import ${pymod}"`  Then we are sure it is using the right interpreter.

Reply:
I like that; good to know for the future. I was running into mismatching pips and pythons on my mac.